### PR TITLE
props updates for 3.1.0

### PR DESCRIPTION
**Pull request by deepchoudhery**
- had to update Microsoft.CodeAnalysis.CSharp.Workspaces to 3.4.0-beta2 as that dependency was required. 
image

### CODE REVIEW

File: version.props [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ﻿<Project>
   <PropertyGroup>
-    <MajorVersion>5</MajorVersion>
-    <MinorVersion>0</MinorVersion>
+    <MajorVersion>3</MajorVersion>
+    <MinorVersion>1</MinorVersion>
     <PatchVersion>0</PatchVersion>
-    <PreReleaseLabel>alpha1</PreReleaseLabel>
+    <PreReleaseLabel>rtm</PreReleaseLabel>
```
Comment (vijayrkn) [inline version.props]: This should be preview1

File: build/dependencies.props [hunk 2]
```diff
@@ -99,7 +99,7 @@
     <MicrosoftEntityFrameworkCoreSqlServerPackageVersion>3.0.0-preview5.19227.1</MicrosoftEntityFrameworkCoreSqlServerPackageVersion>
 
     <!-- Microsoft.Extensions.Configuration.Abstractions -->
-    <MicrosoftExtensionsConfigurationAbstractionsPackageVersion>3.0.0</MicrosoftExtensionsConfigurationAbstractionsPackageVersion>
+    <MicrosoftExtensionsConfigurationAbstractionsPackageVersion>3.1.0-preview1.19506.1</MicrosoftExtensionsConfigurationAbstractionsPackageVersion>
 
     <!-- Microsoft.Extensions.Configuration.EnvironmentVariables -->
     <MicrosoftExtensionsConfigurationEnvironmentVariablesPackageVersion>3.0.0</MicrosoftExtensionsConfigurationEnvironmentVariablesPackageVersion>
```
Comment (vijayrkn) [inline build/dependencies.props]: How about these extensions packages?

Comment (deepchoudhery) [reply build/dependencies.props]: missed "Microsoft.Extensions.Identity.Stores" but only one I could find in the new feed.

File: version.props (hunk 1 above)
Comment (deepchoudhery) [reply version.props]: fixed now

File: build/dependencies.props (hunk 2 above)
Comment (vijayrkn) [reply build/dependencies.props]: @JunTaoLuo - Is there plan to generate 3.1.0 versions for the extensions packages?

**Pull request merged**